package.json

```diff
@@ -8,14 +8,12 @@
     "ios:release:dev": "ENVFILE=.env.development react-native run-ios --mode Release-Dev",
     "ios:release:prod": "ENVFILE=.env.production react-native run-ios --mode Release-Production",
     "ios:qa:distribute": "bundle exec fastlane ios qa_distribute",
-    "ios:testflight:distribute": "bundle exec fastlane ios testflight_distribute",
     "ios:setup": "bundle exec fastlane ios setup_profiles_certs",
     "android:debug:dev": "ENVFILE=.env.development react-native run-android --mode DevelopmentDebug --appIdSuffix=development",
     "android:release:dev": "ENVFILE=.env.development react-native run-android --mode DevelopmentRelease --appIdSuffix=development",
     "android:debug:prod": "ENVFILE=.env.production react-native run-android --mode ProductionDebug --appIdSuffix=production",
     "android:release:prod": "ENVFILE=.env.production react-native run-android --mode ProductionRelease --appIdSuffix=production",
     "android:qa:distribute": "bundle exec fastlane android qa_distribute",
-    "android:google_play:distribute": "bundle exec fastlane android google_play_distribute",
```

Comment:
Why remove these? I think it helps with discoverability

Reply:
They won't work as they require secret information that is on Github. We should only run these through Github anyway